### ISSUE TITLE
Figuring It Out

### ISSUE THREAD

**Issue by princezako** [princezako/boxing-a-cat#1]
We're all a bit unsure on how to proceed, but we'll figure it out!

**Comment by princezako** [comment]
We can watch the videos together after lunch

**Comment by ElizabethBonnett** [comment]
That sounds like a good idea

**Comment by yanakushlova** [comment]
Amazing, let's do this!

**Comment by Callumwren1** [comment]
Amazing Idea 🥇

**Comment by princezako** [comment]
Don't forget to fill in the form so you can get marked

**Comment by YaBoi20035** [comment]
We need to divvy up tasks at some point and in order to do that we need to know what needs to be done.

**Comment by princezako** [comment]
should we watch the videos in 5 mins?

**Comment by ciarancromwell** [comment]
yes i am ready

**Comment by ElizabethBonnett** [comment]
Once someone has finished both of todays notebooks can they let us know and we can then meet to think about how to split tasks

**Comment by yanakushlova** [comment]
That is the plan for the moment, which was decided on our first group meeting, where we watched the tutorials provided to make sure we have everything properly set on GitHub.

**Comment by ElizabethBonnett** [comment]
Everyone has been assigned roles. If there are any issues in the week we can discuss and help/swap where needed.

**Comment by princezako** [comment]
sounds good

**Comment by ElizabethBonnett** [comment]
How is everyone doing with their parts of the project? 

I’ve almost finished figuring out how to upload our package. I just have the last parts of the test upload to do

**Comment by princezako** [comment]
I’m gonna try out some code to actually solve the wave function. I’ll look over the notebook on that to use as a reference point.

**Comment by YaBoi20035** [comment]
Yeah, maybe figure out how the wavefunction solver works and when I push my code you will be able to see how to slot it in.

**Comment by yanakushlova** [comment]
It is going alright, even though I have been sick for a while. I believe tomorrow Ciaran and I will be able to discuss the testing part more in depth and come up with a more detailed plan.

**Comment by ciarancromwell** [comment]
yes i am ready to begin creating a test with yana, we will get started on that today

**Comment by ciarancromwell** [comment]
should we discuss our plan for today in person in 10-15 minutes?

**Comment by ElizabethBonnett** [comment]
Sounds good

**Comment by yanakushlova** [comment]
Yes, Let's to this!

**Comment by princezako** [comment]
yeahhhhhhh

**Comment by ElizabethBonnett** [comment]
Notes from meeting:
Elizabeth will join making tests once she has finished looking at how to upload the package.
Help get the SSH key working
After this we can push and pull the code to look at it and make the tests
Continue with readme

**Comment by ciarancromwell** [comment]
we are going to go through the notebook file for testing to practise in the meantime

**Comment by ciarancromwell** [comment]
@princezako once you have finished writing the input section of the code, please let me know so yana and i can begin testing that section 👍

**Comment by ElizabethBonnett** [comment]
For me to upload the package onto test pypi I will need the project to be finished by early on Thursday at the latest. I hope this is ok with you all

**Comment by princezako** [comment]
I've made some code, just checking if it works with Alex's code

**Comment by Callumwren1** [comment]
Hey guys lets pull some more code to the main github

**Comment by princezako** [comment]
on it

**Comment by yanakushlova** [comment]
After a discussion with team members and Greg we understood well the purpose of the testing. In fact, two separate things should be done to test the code on a separate place and to check the code in the original code, where it is to raise the propper error for the user if needed. The decision to make both testing and checking at the same time was made. Me and Ciaran will split the roles about that and will go through the different stages at the same pace. Once the final code is done, the modifications to the code will be added and the separate testing code file will be ready to run and test.

**Comment by YaBoi20035** [comment]
most of the main code is done now, zack's next pull request will merge it

**Comment by princezako** [comment]
Merged the code into the main branch👌

**Comment by YaBoi20035** [comment]
We're all meeting up from 3:00 to 5:00 pm on the 16th in a group meeting room in the library,

**Comment by princezako** [comment]
we moved to Tolansky because the room was double booked

**Comment by ciarancromwell** [comment]
Hi guys sorry i couldnt meet in person today, im going to focus on getting the testing to work, and hopefully we can do a pull request on someone who is there in person? as i am unable to do so without the correct packages working from here

**Comment by ElizabethBonnett** [comment]
Alex and I (Elizabeth) are in Founders finishing the code if anyone needs us
